ros2_repos_generator.py

```diff
@@ -11,15 +11,28 @@
 """
 
 import argparse
+import getpass
 import json
+import os
 import requests
+from requests.auth import HTTPBasicAuth
 
 api_url = 'https://api.github.com'
 default_ros2_repos = 'https://raw.githubusercontent.com/' \
         'ros2/ros2/master/ros2.repos'
 
 
-def _create_gist(repos_content, anonomous=True):
+def _get_github_token():
+    return os.environ.get('GITHUB_TOKEN')
```

Comment:
I wouldn't create a function for this. First it is only a single trivial line. Second it is only being used in a single location.